interfaces/web.go

```diff
@@ -30,7 +30,7 @@ func (handler WebServiceHandler) CreateConfig(res http.ResponseWriter, req *http
 	server = mainJSON.Server
 
 	res.Header().Set("Content-Type", "application/zip")
-	res.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", "algo"))
+	res.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", "puppet.zip"))
```

Comment:
Wouldn't be better to rename this whit the name of the project? If not, put puppet.zip on a constant so it can be easy to switch in a future.